### PR TITLE
Disabled PLP & PLE components if lack of permissions

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/Atlas/issues/2195

### CODE REVIEW

**Comment by aklochkova** [comment]
@vlbe please note: there are problems with disabling spans with `contenteditable="true"` option

File: js/pages/estimation/components/cca-specification-view-edit.html [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
+                            <button type="button" class="btn btn-primary btn-sm pull-right" data-bind="click: addComparison, disable: !$component.isEditPermitted()"><i class="fa fa-plus"></i> Add Comparison</button>
                             </div>
                             <div class="panel-body">
                                 <faceted-datatable params="{ 
-                                columns: options.cca.comparisonTableColumns, 
-                                options: options.cca.comparisonTableOptions, 
-                                pageLength: options.cca.comparisonTableOptions.pageLength,
-                                lengthMenu: options.cca.comparisonTableOptions.lengthMenu,
-                                dom: options.cca.comparisonTableOptions.dom,
+                                columns: $component.cca.comparisonTableColumns,
```
Comment (wivern) [inline js/pages/estimation/components/cca-specification-view-edit.html:34]: pls, omit the $component prefix when one's not required

File: js/extensions/bindings/contenteditableSwitch.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+define(['knockout'], function (ko) {
```
[the remark this answers is not in the document]
Comment (anton-abushkevich) [reply js/extensions/bindings/contenteditableSwitch.js:1]: That's right, but custom binding looks clearer.

File: js/components/featureextraction/components/covariate-settings-editor.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div data-bind="css: classes()">
+<div data-bind="css: {'covar-settings-editor':true, 'div-disabled':!$component.isEditPermitted()}">
```
[the remark this answers is not in the document]
Comment (anton-abushkevich) [reply js/components/featureextraction/components/covariate-settings-editor.html]: fixed

File: js/pages/estimation/components/cca-specification-view-edit.html (hunk 1 above)
Comment (anton-abushkevich) [reply js/pages/estimation/components/cca-specification-view-edit.html]: fixed

**Pull request merged**